packages/ds-healthcare-gov/src/styles/components/_Footer.scss

```diff
@@ -15,9 +15,9 @@
   padding: var(--spacer-2);
 
   ul[role='list'] {
+    display: flex;
     flex-flow: row wrap;
-    margin: 0;
-    padding: 0;
+    flex-direction: column;
```

Comment:
you can remove the `flex-flow` rule if you're specifying a column direction (the column undoes the flex-flow)

Reply:
Huh, I wonder why I added that. I thought I had tested and fixed the footer. Removing

Reply:
It appears that I still need `flex-flow` since I removed the flex direction. Not sure how that works, but removing it broke it